### PR TITLE
Cpp sdk docs

### PR DESCRIPTION
**Pull request by wmcrobertsq**
Build out of cpp SDK docs.

### CODE REVIEW

**Comment by wmcrobertsq** [comment]
I have read the CLA Document and I hereby sign the CLA

File: content/sdk/quickstarts/cpp/keygendistributed/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Prerequisites
+- A Qrypt Account. [Create an account for free](https://portal.qrypt.com/).
+- Install [CMake](https://cmake.org/).
+- (Windows) Install git bash - comes with typical git install.
+- (Optional) Install [Visual Studio Code](https://code.visualstudio.com/).
+
+## Setup
+1. Clone the repo containing this quickstart to a local folder on a Linux, Mac or Windows platform.
+1. Retrieve a token from the [Qrypt Portal](https://portal.qrypt.com/) and create an environment variable **QRYPT_TOKEN** for it.
+1. Download the Qrypt Security SDK for your platform.
```
Comment (alison-qrypt) [inline content/sdk/quickstarts/cpp/keygendistributed/index.md:21]: It'll be great to have a download link or instruction for it.

File: content/sdk/overview/_index.md [hunk 2]
```diff
@@ -5,20 +5,22 @@ weight = 10
...
-* one-time pads (late 4th Quarter, 2021)
+**Features**
+* local key generation
+* distributed key generation 
 
-Our Quantum Enhanced Classical product line brings value to you by giving you quantum random on your computers, encrypting files with better AES keys generated from quantum random, and better messaging that you're using now because the keys are created from quantum random. But our Everlasting Security product line gives you those things, as well as a seamless upgrade path to Post Quantum security and beyond, when you're ready.
+**Library Compatibility**
 
-{{< figure src="images/crypto-agility-table.svg" >}}
+The C++ SDK is built using the following compilers.
```
Comment (alison-qrypt) [inline content/sdk/overview/_index.md:20]: Curious if it requires exactly the same compilers?

Comment (wmcrobertsq) [reply content/sdk/overview/_index.md:20]: Sometimes yes. Depends on the compiler. So far we haven't had this issue with clang compilations though.

**Review by alison-qrypt — APPROVED** [review]

**Comment by wmcrobertsq** [comment]
recheck

**Review by raul-qrypt — APPROVED** [review]
Looks good

**Comment by alison-qrypt** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by raul-qrypt** [comment]
recheck

**Pull request merged**